sceptre/scipool/config/prod/internalpoolvpc.yaml

```diff
@@ -7,3 +7,4 @@ dependencies:
 parameters:
   VpcSubnetPrefix: "10.41"
   VpcName: internalpoolvpc
+  IncludeBastianSecurityGroup: "false"
```

Comment:
i think we need to keep this because we apply bastian SG to windows instances to allow users to RDP to them after connecting to the VPN.

Reply:
Is this not taken care of by the instance being associated with Vpn security group?

Reply:
yes, you are correct @xschildw .  i confused bastian with vpn SG.

then we'll want to make this change for all VPC files under `sceptre/scipool/config` folder.

Reply:
@zaro0508 the bastian security group is specifically for ssh access from 0.0.0.0: https://github.com/Sage-Bionetworks/aws-infra/blob/master/templates/vpc.yaml#L506-L511, I think you're confusing it with the quarantine security group.

@xschildw do you mean the quarantine security group? they are both being applied, and I'm not sure about the order the security groups are processed in, but even if the quarantine group is applied first, a failure would fall back on the bastian security group which would allow the access.

Reply:
@jesusaurus now that you mention it, the quarantine SGs need to be changed (they're currently referring to 10.1.0.0/16). My understanding was that they should remove all access (i.e. to be used to isolate instances when compromised). Where did you see them being applied (I checked a few instances but no luck...)?

Reply:
@xschildw looking at internalpoolvpc in scipoolprod, we have sg-0bb347378d543521d allowing ssh from the vpn and we also have sg-0ac57202ffc20b43b allowing ssh from 0.0.0.0, is it possible to create a security group that isn't applied? I thought they were automatically applied across the vpc they are associated with

Reply:
@xschildw after doing a bunch of digging, I think you're right, these security groups exist but aren't applied anywhere. but the security group is still a CIS finding and removing it seems like an easy and secure solution.

Reply:
@jesusaurus I would recommend removing it.